classes/Validate.php

```diff
@@ -615,7 +615,7 @@ public static function isEan13($ean13)
      */
     public static function isIsbn($isbn)
     {
-        return preg_match(Tools::cleanNonUnicodeSupport('/^[^<>;={}]*$/u'), $isbn);
+        return !$isbn || preg_match('/^[0-9-]{0,32}$/', $isbn);
```

Comment:
Oh, I did not see that it was for ean13, before, if $isbn was true, isIsbn(false) returned true ?  because, if it was this, we need to keep this behaviour, sorry.. (event if it's not logical)... :-1: